Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -825,11 +828,26 @@ static RefPtr<CSSValue> consumeFontVariantAlternates(CSSParserTokenRange& range)
             if (!ident)
                 return false;
         
-            if (value)
+            hasParsedSomething = true;
+            value = ident->stringValue();
+            return true;
+        };
+        auto parseListAndSetArgument = [&range, &hasParsedSomething] (Vector<String>& value) {
```

Comment:
I think it’s better to use a return value rather than a separate bool and out argument.

We should consider `FixedVector<String>` because that will use less memory after initially creating it. `Vector<String>` is better if we might later append more.